scrapers/POVR.yml

```diff
@@ -0,0 +1,30 @@
+name: POVR
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - povr.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //span[@class="video__quality video__quality--vr mr-3"]/following-sibling::text()
+      Date:
+        selector: //p[contains(@class, 'player__date')]
+        postProcess:
+          - replace:
+              - regex: ^.{1,11}(.*)$
+                with: $1
+          - parseDate: 02 January, 2006
```

Comment:
for the date imo it is better to use a regex bassed on content than positioning
the date in the site does not include 0 padding btw
```yaml
      Date:
        selector: //p[contains(@class, 'player__date')]
        postProcess:
          - replace:
              - regex: .*(\d+\s[a-zA-Z]+,\s\d+)$
                with: $1
          - parseDate: 2 January, 2006
```

Reply:
Done, Told ya I had a Janky date there lol.

Reply:
@JaseNZC i added a last second  \s to the regex before the (
(your merge was faster than my edit :-) )

Reply:
Bam, Lightning fast.